setup.py

```diff
@@ -1,13 +1,26 @@
 from setuptools import setup
 
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
 setup(
     name="pyeafe",
-    version="0.1",
+    version="0.1.1",
     description="Edge-Averaged Finite Elements (EAFE) for FENiCS",
     url="https://github.com/thepnpsolver/pyeafe",
     author="Maximilian Metti, Shuonan Wu, Arthur Bousquet",
-    author_email="pyeafe.team@gmail.com",
+    author_email="bousquet@lakeforest.edu",
     license="GNU Lesser General Public License",
     packages=["pyeafe"],
     zip_safe=False,
+    install_requires=[
+        'fenics>=2019.1.0',
+        'fenics-dolfin>=2019.1.0',
+        'numpy>=1.19.0',
+        'petsc4py=3.12.0'
```

Comment:
this should be: `'petsc4py==3.12.0'`

when I change this, however, it complains that `PETSC_DIR` is not specified...
Petsc is a really annoying package as shown in [their installation help page](https://www.mcs.anl.gov/petsc/petsc4py-current/docs/usrman/install.html).  Notice that the `easy_install` is deprecated in full dedication to making the package suck.